packages/odata-v4/src/de-serializers/default-de-serializers.ts

```diff
@@ -55,7 +56,8 @@ const defaultDeSerializersRaw: DefaultDeSerializers = {
   },
   'Edm.DateTimeOffset': {
     deserialize: deserializeDateTimeOffsetToMoment,
-    serialize: serializeToDateTimeOffset,
+    serialize: (value: moment.Moment, precision?: number) =>
```

Comment:
Interestingly enough, in the diff here in the UI it looks like it's moment.Moment, but you've already made the improvement of only importing { Moment } ...